src/fourc_webviewer/read_geometry_from_file.py

```diff
@@ -146,6 +149,56 @@ def switch_node_order(mesh_exo: Mesh) -> Mesh:
     return copy_mesh_exo
 
 
+def postprocess_exo_mesh(mesh: Mesh) -> Mesh:
+    """Postprocessing steps for the read-in Exodus mesh.
+
+    Args:
+        mesh: read-in Exodus mesh
+    Returns:
+        Mesh: postprocessed read-in Exodus mesh
+    """
+
+    return switch_node_order(mesh_exo=mesh)
+
+
+def postprocess_vtu_mesh(mesh: Mesh) -> Mesh:
+    """Postprocessing steps for the read-in vtu mesh.
+
+    Args:
+        mesh: read-in vtu mesh
+    Returns:
+        Mesh: postprocessed read-in vtu mesh
+    """
+    copy_mesh = mesh.copy()
+
+    # --> loop through point sets in point_data, and move them to dedicated point_sets "point_set_1" -> "1"
+
+    # collect point data keys to move
+    keys_to_rename = []
+    for pd_array in copy_mesh.point_data:
+        if pd_array.startswith("point_set_"):
+            keys_to_rename.append(pd_array)
+
+    # move "point data" to the dedicated point sets
+    for old_key in keys_to_rename:
+        new_key = old_key.replace("point_set_", "")
+        copy_mesh.point_sets[new_key] = np.where(mesh.point_data[old_key] == 1)[0]
+        del copy_mesh.point_data[old_key]
```

Comment:
I think this is a dictionary, and the pop method should be used instead. Normally it's not desired in Python to use `del`

```suggestion
        copy_mesh.point_data.pop(old_key)
```